test_lsf_can_retrieve_stdout_and_stderr fails

After the upgrade endeavours this test fails, very likely related to the changes made.

```
        if tail_chars_to_read > num_written_characters:
            assert f"LSF-stderr:\n{_err}" in message
            # we get some extra echos after LSF-out
>           assert f"{_out}" in message
E           AssertionError: assert 'yJCgZZrWAIqpgISeraDymqJBbvrbfdlCpakprToJsmjMStysNGcjxtRNVNHFXqOOnropSnRWFiOQZLRspxmYSyCODeeeJamjeyrdHJFljUcBIlIlLKqoxHrFeZeFqwFnLYDPEJAVwkvYFozJnLvnJglGYHHVYVkhjyhkXhqTJjfkRBeLZGjOQikMMHVPpPLXKDXsybfVbfyhWBjNbtnNPlbueHgXnkBHlImZYTYhDUkYKUnSDByTajFOpQGAxrUpdDSFhxyxvLcynMBFSSUBKfrIQjGKtbVilUQDFyeQSTlQbebsjmYSJMPoYOzIkfTFXcNwwslOJpYveoUewOwtyMiZeUNlaAXdcRMEgCWcdQdbMxzHcOGXLnYJVtDRBpzvjlGKIfLnQNRnbUQPRPdsXhUzKrKKsZBmKfjyfmDEAXsMfWFIxqJUONbBmBWJEWgKRoLQACJicOaSHefyMGqbspLmlasFecEstGgSflzuUKYsFTqPwtzZtpUwqhjeEtlWaDjxtbRbGhnsSiHgMgRTLMsDQdzbwtbTxyEqRqpCUCjauoSBdCRNrDXHTIWXKxPcEsVCxBMbLKASGVtUQGgrUhkU' in '\n    LSF-stderr:\nAlkjPYBAcFsfNHGbSwMGNXbpcRXKKnOLCbHorkcaKQyvSGufcAMiTLXrewVVUXrWWtTDvCgCLeYGSSYaJiXRQsURAthmlAtHhkilYIBbRiUlJLKfHFPEbnPHWWeTNWlKqCcRguKxqlgfbmXJcYBFgwYAmhbTyuPYMpdoQUYsDPEdFzZhdRSVqquMuYZRuGLlKBHXeTDYXVMhRJxUimmukkJtsOpGqSqhvLXTvqQRPPpkwbkysCoFzjFWxMoiRhfskVgyUyyozhuFLZFxWqvNZwxfxziLznXVwTIInoKRVXvcgAXhoemwtPMjsirRptMDoCshrKeEAzLsoymMrbKbKoFOlILhCcyXmMqUXRUXciHqgSiVUKOZEkULGcpdFIvJypEBwlgIgdmkVbspePZcILQxQjMSJgrhhyzmUzOBdnWfqBxPyJuBmncmljAvwtlMEfGkcqIbNIzpLfBquZWRkcZrDGfgpwuDlfxWyNJUWQTcbxXCztUxnfaXWZMAvSVTPkxIymnGDcMSOtdaLaRdKvHHMDYdfUVDOGsWzBOzpkZTQsywAcbWUfBbfOvKAIyKUsdAsqTxjeymzVOcqDsUReqr\n\n    LSF-stdout:\nxmYSyCODeeeJamjeyrdHJFljUcBIlIlLKqoxHrFeZeFqwFnLYDPEJAVwkvYFozJnLvnJglGYHHVYVkhjyhkXhqTJjfkRBeLZGjOQikMMHVPpPLXKDXsybfVbfyhWBjNbtnNPlbueHgXnkBHlImZYTYhDUkYKUnSDByTajFOpQGAxrUpdDSFhxyxvLcynMBFSSUBKfrIQjGKtbVilUQDFyeQSTlQbebsjmYSJMPoYOzIkfTFXcNwwslOJpYveoUewOwtyMiZeUNlaAXdcRMEgCWcdQdbMxzHcOGXLnYJVtDRBpzvjlGKIfLnQNRnbUQPRPdsXhUzKrKKsZBmKfjyfmDEAXsMfWFIxqJUONbBmBWJEWgKRoLQACJicOaSHefyMGqbspLmlasFecEstGgSflzuUKYsFTqPwtzZtpUwqhjeEtlWaDjxtbRbGhnsSiHgMgRTLMsDQdzbwtbTxyEqRqpCUCjauoSBdCRNrDXHTIWXKxPcEsVCxBMbLKASGVtUQGgrUhkU\n\n\nPS:\n\nRead file </private/f_scout_ci/pytest-tmp/tmp.paqEZOKW4A/test_lsf_can_retrieve_stdout_a3/test_lsf_can_retrieve_stdout_and_stderr.LSF-stderr> for stderr output of this job.\n\n'
```

https://github.com/equinor/komodo-releases/actions/runs/9247298540/job/25436442415

--- a/tests/integration_tests/scheduler/test_lsf_driver.py
+++ b/tests/integration_tests/scheduler/test_lsf_driver.py
@@ -92,15 +92,11 @@ async def test_lsf_can_retrieve_stdout_and_stderr(
         num_characters_to_read_from_end=tail_chars_to_read,
     )
 
-    if tail_chars_to_read > num_written_characters:
-        assert f"LSF-stderr:\n{_err}" in message
-        # we get some extra echos after LSF-out
-        assert f"{_out}" in message
-    else:
-        assert f"LSF-stderr:\n{_err}" not in message
-        assert f"LSF-stderr:\n{_err[-tail_chars_to_read+1:]}" in message
-        assert f"LSF-stdout:\n{_out}" not in message
-        assert f"LSF-stdout:\n{_out[-tail_chars_to_read+1:]}" in message
+    stderr_txt = Path(f"{job_name}.LSF-stderr").read_text(encoding="utf-8").strip()
+    stdout_txt = Path(f"{job_name}.LSF-stdout").read_text(encoding="utf-8").strip()
+
+    assert stderr_txt[-min(tail_chars_to_read, num_written_characters) + 1 :] in message
+    assert stdout_txt[-min(tail_chars_to_read, num_written_characters) + 1 :] in message
 
 
 async def test_lsf_cannot_retrieve_stdout_and_stderr(tmp_path, job_name):

